consensus/src/units/store.rs

```diff
@@ -28,6 +36,25 @@ impl<'a, H: Hasher, D: Data, KB: KeyBox> UnitStore<'a, H, D, KB> {
         }
     }
 
+    pub fn get_status(&self) -> UnitStoreStatus {
+        let mut top_row: Vec<(usize, Round)> = self
+            .by_coord
+            .keys()
+            .map(|c| (c.creator, c.round))
+            .into_group_map()
+            .into_iter()
+            .map(|(k, v)| (k, v.into_iter().max().unwrap_or(0)))
+            .map(|(k, v)| (k.0, v))
+            .collect();
+        top_row.sort();
```

Comment:
This computes "highest unit per node index", apart from that it would be good to signal if some unit is missing, i.e., suppose that we hold (for a specific node index) units from rounds `0, 1, 2, 4, 5, 6, 7, 8, 10` then the above will just tell us `10` -- but I would also like to know that `3` is the earliest round that we are missing. This information can be printed only if there is a "hole", otherwise we can skip it. 

Another thing is this `.unwrap_or(0)` -- we don't differentiate between the `0`th unit and no units?

Reply:
0 was just for safety :P and I use `Vec<(usize, Round)>` (for better readability), so in case of no units there will be no entry

Reply:
added holes